scripts/update-dockerhub-description.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+image_name="all-in-one"
+readme_path="./README.md"
+token="$DOCKERHUB_TOKEN"
+repository="jaegertracing/$image_name"
+url="https://hub.docker.com/v2/repositories/$repository/"
```

Comment:
We upload images to both Docker Hub and quay.io. Does the latter also have an API to upload README?